Selenium.Axe/Selenium.Axe/AxeRunOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Selenium.Axe
+{
+    [JsonObject]
+    public class RunOnlyOptions
```

Comment:
Do we expect a consumer of the library to ever want to reference these new classes directly? If they're not intended to be part of the public API surface, consider making them `internal` and using `InternalsVisibleTo` to enable unit tests.

Reply:
RunOptions is exposed in AxeResult. but i can make RunContext internal. (axe doesnt expose that in result